src/contracts/pods/EigenPodManagerStorage.sol

```diff
@@ -64,6 +64,8 @@ abstract contract EigenPodManagerStorage is IEigenPodManager {
      */
     mapping(address => int256) public podOwnerShares;
 
+    uint64 public constant denebForkTimestamp = 1705473120;
```

Comment:
copy-pasting my comment from elsewhere:
just validated this myself
Teku has the Goerli fork epoch as 231680 here https://github.com/Consensys/teku/pull/7857/files#diff-dc8fb07d95769aadd81ea69e5f034ece1fd19cd7803a09042d08bdb20483cd66R45 and 7413760 is the first slot of this epoch, which I checked with the [beaconcha.in](http://beaconcha.in/) explorer (https://goerli.beaconcha.in/slot/7413760 vs https://goerli.beaconcha.in/epoch/231679)
and `1705473120` is indeed the timestamp for slot `7413760`